resources/default.rb

```diff
@@ -34,6 +35,16 @@
     uid new_resource.nexus3_uid unless new_resource.nexus3_uid.nil?
   end
 
+  new_resource.nexus3_additional_groups.to_a.each do |gr|
+    group "add #{new_resource.nexus3_user} user to #{gr} group" do
+      group_name gr
+      action :manage
```

Comment:
I am not sure we want to fail, that's why I used manage in the first place, but yeah I am divided between it will show there is an issue and does it matter ?
And it is like I just want to "manage" members, not really modify the group more than that.